data/Audiences.ts

```diff
@@ -2,30 +2,30 @@ import { Audience } from "../types/targeting";
 
 const audiences: { [id: string]: Audience } = {
   all_english: {
-    name: "All English users (release)",
+    name: "All English users",
     description: "All users in en-* locales using the release channel.",
-    targeting: "localeLanguageCode == 'en' && browserSettings.update.channel == 'release'",
```

Comment:
If we want to filter based on a choice in the experimenter ui, we need to keep this logic in and just change `'release'` to `{firefox_channel}`